config.json

```diff
@@ -25,34 +25,34 @@
         "slug": "hello-world",
         "name": "Hello World",
         "uuid": "00c787bd-8f2d-4759-b87c-e0ff65c5989d",
-        "practices": [],
+        "topics": [],
```

Comment:
The empty `topics` fields can be removed, as we don't do anything with them.

Reply:
Good to know, but if I'm planning on just adding to them soon then it seems it should be ok to leave them in place also.